pinterest_export/scraper.py

```
@@ -0,0 +1,112 @@
+        for selector in [
+            "button:has-text('Accept')", "button:has-text('accepteer')",
+            "button:has-text('I agree')", "button:has-text('Akkoord')",
+            "button:has-text('Continue')",
+        ]:
+            try:
+                btn = page.locator(selector).first
+                if await btn.is_visible(timeout=2000):
+                    await btn.click()
+                    break
+            except Exception:
+                continue
```

Ruff (0.15.2)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)